### ISSUE TITLE
A11y_Model Builder_Add data_Advanced data option_Info and relationship: After selecting any list item from dropdown then narrator is announcing previous selected list item details instead of updated.

### ISSUE THREAD

**Issue by SachinChavhan317** [dotnet/machinelearning-modelbuilder#2677]
### GitHubTags:
#WCAG1.3.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Info and Relationships; #Screenreader; #Narrator; #FTP; #Win32; #Benchmark;

### Actual Result:
After selecting any list item from dropdown then narrator is announcing previous selected list item details instead of updated.

This issue observed throughout the application wherever the dropdown is present.

Issue not reproducible with NVDA.

### Expected Result:
After selecting any list item from dropdown then narrator should be announce updated list item details instead of previous one.

### Environment Details:
Application Name: Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: Narrator
NVDA version: 2020.4

### Repro Steps:

1. Turn on Narrator.
2. Launch the Visual Studio with at least one of the .NET workload.
3. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
4. Create a new .NET project or open an existing .NET project.
5. Open context menu(Right-click) of the project in the solution explorer.
6. Tab till Add and then tab till Machine Learning Model and press enter.
7. Model Builder dialog will open. Complete Scenario and Environment setup steps.
8. Tab till value prediction tiles and press enter.
9. Add Data screen will appear. Tab till Advance data option and press enter.
10. Verify whether After selecting any list item from dropdown then narrator is announcing updated list item or not.

Verify whether the narrator is reading the tab items twice or not.
### User Impact:
Screen reader user will not be able to get the proper information regarding the tab items.

### Attachment:
Narrator:
![Presentation3](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/b197b6b9-ff31-47e7-828e-ce1eee98b06c)

NVDA:
![Presentation2](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/4078390c-6886-4e9e-8880-6619b8138704)

**Comment by zewditu** [comment]
The issue is not reproduced consistently, it seems that there is a time delay between the UI update and the narrator's announcement.

**Comment by zewditu** [comment]
@SachinChavhan317 This is not reproducible with NVDA and  it seems that the narrator also narrate selected item sometimes hence it appears that the issue is specific to the narrator 
are you able to down the sev of this issue?

**Comment by SachinChavhan317** [comment]
@zewditu We cannot downgrade the severity for this issue. After selecting new item form the list, screen reader is announcing previous list item name even the selected list item got updated.

We can downgrade this bug to sev3, if screen reader doesn't announce previous selected list item name once new list item get updated. Thanks!

**Comment by zewditu** [comment]
And I also noticed that the issue is not reproduced if the user selects the item using the mouse.

**Comment by zewditu** [comment]
@SachinChavhan317  I think we need to close this issue, as per our discussion during accessibility office hours on 7/18/2023 with experts it is because of this known issue https://microsoft.visualstudio.com/OS/_workitems/edit/38960730.

**Comment by SachinChavhan317** [comment]
As per discussion with SME, this is a narrator issue. Hence closing the bug.

**Comment by SachinChavhan317** [comment]
#Regressed:07-19-23; #Closed;
